assets/js/modules/sign-in-with-google/components/settings/SettingsView.js

```diff
@@ -147,7 +147,7 @@ export default function SettingsView() {
 				<div className="googlesitekit-settings-module__meta-item">
 					<h5 className="googlesitekit-settings-module__meta-item-type">
 						{ __(
-							'Anyone can register WP setting',
+							'Users can create new accounts',
```

Comment:
This should match the change above :)
```suggestion
							'User registration',
```